modules/odc-importing-codebase-from-git-to-create-application.adoc

```diff
@@ -15,7 +15,25 @@ The following procedure walks you through the *From Git* option in the *Develope
 * *Context Dir* to specify the subdirectory for the application source code you want to use to build the application.
 * *Source Secret* to create a *Secret Name* with credentials for pulling your source code from a private repository.
 
-. In the *Builder* section, after the URL is validated, an appropriate builder image is detected, indicated by a star, and automatically selected. For the `https://github.com/sclorg/nodejs-ex` Git URL, the Node.js builder image is selected by default. If a builder image is not auto-detected, select a builder image. If required, you can change the version using the *Builder Image Version* drop-down list.
+. Optional: You can import a Devfile or a Dockerfile through your Git repository to further customize your deployment.
+Devfile::
+* If your Git repository contains a Devfile, it is automatically detected and populated on the *Devfile path* field.
+* To edit the file import type and select a different strategy, click *Edit import strategy* option.
+* If multiple Devfiles are detected, the Devfile path needs to be specified, relative to the *Context Dir* field, to import a specific Devfile.
+
+Dockerfile::
+* If your Git repository contains a Dockerfile, it is automatically detected and populated on the *Dockerfile path* field.
+* To edit the file import type and select a different strategy, click *Edit import strategy* option.
+* If multiple Dockerfiles are detected, specify the path of the specific Dockerfile relative to the *Context Dir* field.
+
+Devfile and Dockerfile::
+* If a Devfile and a Dockerfile are detected in the same repository, the Devfile is selected by default. To choose a different strategy, click *Edit import strategy*.
+* If multiple Devfiles and Dockerfiles are detected in the same repository, you can specify which file you wish to import.
```

Comment:
The information is highly repeated, can we consolidate it all like this:
1. If your Git repository contains a Devfile or a Dockerfile, it is automatically detected and populated on the Devfile path or the  Dockerfile path field, respectively. If a Devfile and a Dockerfile are detected in the same repository, the Devfile is selected by default.
2. To edit the file import type and select a different strategy, click Edit import strategy option.
3. If multiple Devfiles or Dockerfiles are detected, the Devfile or Dockerfile path needs to be specified, relative to the Context Dir field, to import a specific Devfile or Dockerfile.